features/atomic/atomic_mount_unmount.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Atomic mount and unmount test for new upgrade tree
+    Describes tests for mount/umount to new ostree
+
+Background: Atomic hosts are discovered
+      Given "cihosts" hosts from dynamic inventory
```

Comment:
@miabbott, agree, it's just quickly debug with cihosts in here, okay, I will update v2 patch.